gi-gtk-declarative-app-simple/test/Main.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE OverloadedLists  #-}
+module Main where
+
+import           Control.Monad                 (void)
+import qualified GI.Gtk                        as Gtk
+import           GI.Gtk.Declarative
+import           GI.Gtk.Declarative.App.Simple
+import           Pipes
+import           System.Timeout
+import           Test.Hspec
+
+main :: IO ()
+main = hspec $
+  describe "run" $ do
+    it "propagates exceptions from view function" $
```

Comment:
Looks good :+1: 

I have now thought of another test: what happens if `update'` returns a `Transition s cmd` where `cmd` throws an exception?

Reply:
I do think that right now it will be fine because there is no more `forkIO` (so no more ignored exception) or no more `threadsAddIdle`, so no more exception in an uncontrolled thread. But this is worth a test.